data/pigui/modules/radar.lua

```diff
@@ -269,10 +354,10 @@ radar3d.draw = function(self, center)
 	end
 end
 
--- This variable needs to be outside the function in order to capture state
--- between frames. We are trying to ensure that a mouse right-click started and
--- finished inside the radar area in order to trigger the popup.
-local click_on_radar = false
+local function onTargetClicked(target)
+	-- TODO: Should ships be set as nav or combat targets?
+	Game.player:SetNavTarget(target.body)
```

Comment:
Ships are always combat targets.

Reply:
> Ships are always combat targets.

Not really - rescue missions, for example.

Reply:
Fixed in 0665cc57934c34ee990dd7c0eedabe5691bce352

Reply:
Does the ship you've targeted as a "Combat Target" instead of as a "Nav Target" "know" how you targeted it?

Ie, does it get an alert "weapons lock" and go into combat-mode itself? Or is this purely a distinction in the player ships' HUD/systems?

Reply:
Not yet. This is something I want to dive into more deeply in a sensors rework - adding different types of passive information your own sensors can pick up (radar emissions, transponder signals, ship drive plumes, IR/EM flares from missile launches, etc), and giving NPC ships an understanding of them.

With that rework, nav targets would likely be something your computer maintains passively from sensor inputs, with a separate class of targets that your active sensors are maintaining a "trackfile" on, and finally what we currently call the combat target being the primary trackfile on which your sensors are providing good position, velocity, and aspect estimations for firing solutions.

I would also like it to tie into a rework of missiles and countermeasures, introducing different guidance methods and corresponding countermeasure methods to attempt to defeat missile guidance and enemy sensors.

...but that's a topic for another day. Feel free to add this to the dev docs (with your own thoughts on the matter) if you so desire.